javascript/callOnce.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Closure
+ *
+ * @param  {function} callback, fn
+ * @return {function}
+ */
+function callOnce(callback) {
+
+    var isCalled = 0;
```

Comment:
Making it `0` and `1` is very odd in JavaScript world -- sorry if i was unclear on the call. `false` and `true` are the way to go.